src/test/jest/extract.test.js

```diff
@@ -4,45 +4,75 @@ const sizeof = require('object-sizeof');
 
-describe('Feature extraction tests', () => {
-    beforeEach(() => {
-        // Nothing yet
-    });
+    const dumpMeta = {
+        endpointId: '1a404b1b',
+        dumpPath: dumpPath,
+        statsFormat: statsFormat
+    };
 
-    test('SFU dump feature extraction', async () => {
-        const dumpMeta = {
-            endpointId: '1a404b1b',
-            dumpPath: './src/test/jest/sfu',
-            statsFormat: StatsFormat.CHROME_STANDARD
-        };
+    const featExtractor = new FeatureExtractor(dumpMeta);
+    const results = await featExtractor.extract();
 
-        const featExtractor = new FeatureExtractor(dumpMeta);
+    const resultString = fs.readFileSync(resultPath);
+    const resultList = JSON.parse(resultString);
+    const resultTemplate = resultList.shift();
 
-        const results = await featExtractor.extract();
+    assert.deepStrictEqual(results, resultTemplate.features);
+}
 
-        logger.info('%o', results);
+describe('Feature extraction tests', () => {
 
-        expect(3).toBe(3);
+    test.skip('SFU dump feature extraction', async () => {
+        await simulateConnection(
+            './src/test/jest/sfu',
+            '',
+            StatsFormat.STANDARD
+        );
     });
 
     test.skip('SFU and P2P dump feature extraction', async () => {
-        const dumpMeta = {
-            endpointId: '1a404b1b',
-            dumpPath: './src/test/jest/sfu-p2p'
-        };
-        const featExtractor = new FeatureExtractor(dumpMeta);
+        await simulateConnection(
+            './src/test/jest/sfu-p2p',
+            '',
+            StatsFormat.STANDARD
+        );
+    });
 
-        try {
-            const results = await featExtractor.extract();
+    test('Chrome in a peer-to-peer call', async () => {
+        await simulateConnection(
+            './src/test/dumps/google-standard-stats-p2p',
+            './src/test/jest/results/google-standard-stats-p2p-result.json',
+            StatsFormat.CHROME_STANDARD
+        );
+    });
 
-        } catch (e) {
-            logger.error('Error: %o', e);
-        }
+    test('Chrome in a multi-party call', async () => {
+        await simulateConnection(
+            './src/test/dumps/google-standard-stats-sfu',
+            './src/test/jest/results/google-standard-stats-sfu-result.json',
+            StatsFormat.CHROME_STANDARD
+        );
+    });
 
+    test('Firefox in a multi-party call', async () => {
+        await simulateConnection(
+            './src/test/dumps/firefox-standard-stats-sfu',
+            './src/test/jest/results/firefox-standard-stats-sfu-result.json',
+            StatsFormat.FIREFOX
+        );
+    });
 
-        expect(3).toBe(3);
+    test('Safari in a peer-to-peer call', async () => {
+        await simulateConnection(
+            './src/test/dumps/safari-standard-stats',
+            './src/test/jest/results/safari-standard-stats-result.json',
+            StatsFormat.SAFARI
+        );
     });
 });
```

Comment:
Is it on purpose that you convert the newer tests to the new jest result format, but then don't run them in here?

If not I can take care of adding these in a later PR.